grammar.go

```diff
@@ -452,7 +454,38 @@ func (r *Grammar) ModifyDefault(blueprint driver.Blueprint, column driver.Column
 	return ""
 }
 
-func (r *Grammar) ModifyNullable(blueprint driver.Blueprint, column driver.ColumnDefinition) string {
+func (r *Grammar) ModifyGeneratedAs(_ driver.Blueprint, column driver.ColumnDefinition) string {
+	if column.IsSetGeneratedAs() {
```

Comment:
Optimizing the structure would be graceful.

```suggestion
	if !column.IsSetGeneratedAs() {
		return ""
	}
```